packages/components/addon/components/hds/stepper/indicator/step.hbs

```diff
@@ -1,6 +1,6 @@
 <div class={{this.classNames}} ...attributes role="listitem" aria-label={{@status}}>
```

Comment:
And consequently an `<li>` here?

Reply:
@MelSumner @alex-ju why are we making this so complex? why don't just remove the `role="listitem"` here? this is not a list item, and if we remove it we can avoid adding a parent wrapper, and all this "problem" goes away.
The indicator should be evalutated in the context of the whole stepper, is a sub-sub-component.
<img width="1025" alt="screenshot_1737" src="https://user-images.githubusercontent.com/686239/181786679-23df633e-ccf8-47f5-b27f-d101758e6c6d.png">
/cc @jorytindall

Reply:
Looking at the diagram, yes `<Hds::Stepper:Step>` is an item, but the indicator isn't. So, I'm happy to remove the roles we defined and postpone that for when we tackle the `<Hds::Stepper:Step>`

Reply:
I admit, I've not checked the CRD/component anatomy first and got confused by the presence of these `role` attributes.

Reply:
> why are we making this so complex?

I think it would be good for you to clarify your claim that this PR is adding complexity.  

It is doing the opposite. It's adding a single template-only component that provides a small, flexible wrapper that will help users meet WCAG conformance requirements. 

After looking at this screenshot, I'm even more convinced that there is no complexity being added. It's both flexible and future-proof in implementation.